### DISCUSSION TITLE
Polybar disabling ipc channels due to error : permission error

### DISCUSSION THREAD

**Discussion by January180** [polybar/polybar#2786] [Q&A] [answered]
the error in question : "error: Disabling IPC channels due to error: /usr/src/debug/polybar-3.6.3/src/components/eventloop.cpp:112: libuv error for 'uv_pipe_bind(get(), path.c_str())': permission denied"

 it happens when i run "polybar" as user in my terminal alacritty  but it doesnt seem to stop polybar from running as a whole

I've looked at the recommended boards and troubleshooting guides but cant find anything related to directly what I'm facing.

I'm trying to remove this error as a whole but I'm not quite sure on what I'd do because /usr/src/debug/polybar-3.6.3/src/components/ is not an actual directory on my system

I tried to change the permissions of this directory to user instead of root but as said above it does not exist, polybar-msg responds back with "polybar-msg: No active ipc channels"

Polybar ver : 3.6.3
WM : kwin
I start polybar by typing "polybar" in terminal

[config.txt](https://github.com/polybar/polybar/files/9378311/config.txt) not sure on what would be relevant in the config file, sorry

sorry if this is obvious! very new to linux and I've looked as far as I have had time for this particular issue :frowning_face:

**Comment by patrick96** [comment, marked as answer]
I don't think you're doing anything wrong. This is an internal error and likely caused by some kind of bug.

Polybar still continues to run, the error just disable IPC functionality.

Could you run polybar with trace logging (`polybar -l trace ...`) and post the entire output of that? This may give us a better insight into how this happened.

**Reply by January180** [reply]
its fine now, i had to do a full reinstall of manjaro because i messed up some of the core files, thank you for the response

**Comment by January180** [comment]
zazed
